include/zephyr/drivers/clock_control/clock_control_ambiq.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023 Ambiq Micro Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_CLOCK_CONTROL_CLOCK_CONTROL_AMBIQ_H_
+#define ZEPHYR_INCLUDE_DRIVERS_CLOCK_CONTROL_CLOCK_CONTROL_AMBIQ_H_
+
+#include <zephyr/device.h>
+#include <zephyr/sys/onoff.h>
```

Comment:
all includes are redundant.

Reply:
@nordic-krch, thanks for your review. I clean up the redundant codes.